swagger.yaml

```diff
@@ -70,7 +70,7 @@ paths:
           type: string
         - in: query
```

Comment:
I thought about this during the implementation. I went with just allowing a single type to make the implementation a bit simpler. For now, if needed the consumer can make multiple API calls. Down the road once we migrate to using an ES library, if needed we can support multiple type parameters.